_posts/2025-02-03-Reduce-Cost-with-Disk-based-Vector-Search.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+title:  "Reduce costs with disk-based vector search"
+authors:
+   - jmazane
+   - vikash
+   - vamshin
+   - kolchfa
+date: 2025-02-03
+has_science_table: true
+categories:
+  - technical-posts
+meta_keywords: Disk-based vector search in OpenSearch, quantization, ANN search, compression, performance testing, binary quantization
+meta_description: Learn how to reduce your OpenSearch costs for vector search using the new disk-based vector search.
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ disk based vector search, vector search, vector quantization techniques, OpenSearch 2.17, binary quantization, natural language embedding models

_meta_description:_ Learn how OpenSearch significantly reduces vector search costs using disk-based methods. Discover quantization techniques and practical implementation strategies for efficient large-scale vector search operations.